test/refactoring/AddCodableImplementation/Outputs/enum/codable.swift.expected

```diff
@@ -0,0 +1,65 @@
+
+enum Payload: Codable {
+  case plain(String)
+  case pair(key: String, value: String)
+
+  enum CodingKeys: CodingKey {
+    case plain
+    case pair
+  }
+  enum PlainCodingKeys: CodingKey {
+    case _0
+  }
+  enum PairCodingKeys: CodingKey {
+    case key
+    case value
+  }
+
+  init(from decoder: Decoder) throws {
+    let container = try decoder.container(keyedBy: Payload.CodingKeys.self)
+
+    guard container.allKeys.count == 1 else {
+      throw DecodingError.typeMismatch(Payload.self, DecodingError.Context.init(codingPath: container.codingPath, debugDescription: "Invalid number of keys found, expected one.", underlyingError: nil))
+    }
+    switch container.allKeys.first.unsafelyUnwrapped {
+    case .plain:
+
+      let nestedContainer = try container.nestedContainer(keyedBy: Payload.PlainCodingKeys.self, forKey: Payload.CodingKeys.plain)
+
+      self = Payload.plain(try nestedContainer.decode(String.self, forKey: Payload.PlainCodingKeys._0))
+      case .pair:
```

Comment:
The indentation is a little weird here, but I don't think that's super important.

Reply:
Seems like it wouldn't be *too* hard to fix but... 🤷 See above, I wouldn't have bothered at all 😅